src/Backend/Modules/MediaLibrary/Js/MediaLibraryHelper.js

```diff
@@ -841,15 +1032,9 @@ jsBackend.mediaLibraryHelper.upload =
             validation: {
                 allowedExtensions: jsBackend.data.get('MediaLibrary.mediaAllowedExtensions')
             },
-            scaling: {
-                includeExif: true,
-                sendOriginal: false,
-                sizes: [
-                    {name: "", maxSize: 3000}
```

Comment:
Why exactly?
1 will give lots of user errors, since I tested this for a while and 3000 was a nice compromise between, working images and failing images...